projects/1.18/assets/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,201 @@
+  "item.butchercraft.cooked_ground_pork": "熟猪绞肉",
+  "item.butchercraft.cooked_heart": "熟心",
+  "item.butchercraft.cooked_kidney": "熟肾",
+  "item.butchercraft.cooked_lamb_rib": "熟羊肋条",
+  "item.butchercraft.cooked_lamb_ribs": "熟羊排骨",
+  "item.butchercraft.cooked_lamb_roast": "熟的大块羊肉",
+  "item.butchercraft.cooked_lamb_scraps": "熟羊肉丝",
+  "item.butchercraft.cooked_lamb_stewmeat": "熟羊碎肉",
+  "item.butchercraft.cooked_liver": "熟肝",
+  "item.butchercraft.cooked_lung": "熟肺",
+  "item.butchercraft.cooked_oxtail": "熟牛尾",
+  "item.butchercraft.cooked_pork_rib": "熟猪肋条",
+  "item.butchercraft.cooked_pork_ribs": "熟猪排骨",
+  "item.butchercraft.cooked_pork_roast": "熟的大块猪肉",
+  "item.butchercraft.cooked_pork_scraps": "熟猪肉丝",
+  "item.butchercraft.cooked_pork_stewmeat": "熟猪碎肉",
+  "item.butchercraft.cooked_sausage": "熟香肠",
+  "item.butchercraft.cooked_stomache": "熟肚",
+  "item.butchercraft.cooked_tongue": "熟口条",
+  "item.butchercraft.cooked_tripe": "熟肠子",
+  "item.butchercraft.cow_carcass": "待宰硕牛",
+  "item.butchercraft.cubed_beef": "生牛肉丁",
+  "item.butchercraft.cubed_lamb": "生羊肉丁",
+  "item.butchercraft.cubed_pork": "生猪肉丁",
+  "item.butchercraft.fat": "脂肪",
+  "item.butchercraft.fried_brains": "炸脑花",
+  "item.butchercraft.fried_chicken": "炸鸡块",
+  "item.butchercraft.fried_fish": "炸鱼",
+  "item.butchercraft.fries": "薯条",
+  "item.butchercraft.gelatin": "吉利丁",
+  "item.butchercraft.gravy_boat_item": "肉汁",
+  "item.butchercraft.grilled_cheese": "烤奶酪三明治",
+  "item.butchercraft.ground_beef": "生牛绞肉",
+  "item.butchercraft.ground_lamb": "生羊绞肉",
+  "item.butchercraft.ground_pork": "生猪绞肉",
+  "item.butchercraft.gut_knife": "穿肠刀",
+  "item.butchercraft.haggis": "碗装哈吉斯",
+  "item.butchercraft.hash": "碗装肉丁土豆",
+  "item.butchercraft.heart": "生心脏",
+  "item.butchercraft.horn": "角",
+  "item.butchercraft.jelly_black": "碗装黑莓果冻",
+  "item.butchercraft.jelly_blue": "碗装蓝莓果冻",
+  "item.butchercraft.jelly_brown": "碗装可乐果冻",
+  "item.butchercraft.jelly_cyan": "碗装潘趣果冻",
+  "item.butchercraft.jelly_green": "碗装苹果果冻",
+  "item.butchercraft.jelly_grey": "碗装神秘果冻",
+  "item.butchercraft.jelly_light_blue": "碗装蓝色覆盆子果冻",
+  "item.butchercraft.jelly_light_grey": "碗装火龙果果冻",
+  "item.butchercraft.jelly_lime": "碗装酸橙果冻",
+  "item.butchercraft.jelly_magenta": "碗装杂莓果冻",
+  "item.butchercraft.jelly_orange": "碗装橘子果冻",
+  "item.butchercraft.jelly_pink": "碗装草莓果冻",
+  "item.butchercraft.jelly_purple": "碗装葡萄果冻",
+  "item.butchercraft.jelly_red": "碗装樱桃果冻",
+  "item.butchercraft.jelly_white": "碗装椰子果冻",
+  "item.butchercraft.jelly_yellow": "碗装柠檬果冻",
+  "item.butchercraft.ketchup_jar_item": "番茄酱",
+  "item.butchercraft.kidney": "生肾脏",
+  "item.butchercraft.lamb_jerky": "羊肉干",
+  "item.butchercraft.lamb_rib": "生羊肋条",
+  "item.butchercraft.lamb_ribs": "生羊排骨",
+  "item.butchercraft.lamb_roast": "生的大块羊肉",
+  "item.butchercraft.lamb_scraps": "生羊肉屑",
+  "item.butchercraft.lamb_stew": "炖羊肉",
+  "item.butchercraft.lamb_stewmeat": "生羊碎肉",
+  "item.butchercraft.lard": "大油",
```

```suggestion
  "item.butchercraft.lard": "油脂",
```
因为“大油”“猪油”等全都是特指某一种动物油，但本模组内3种动物都能产出“fat”，所以用更广泛的名称